commandapi-platforms/commandapi-bukkit/commandapi-bukkit-nms/commandapi-bukkit-1.21.4/src/main/java/dev/jorel/commandapi/nms/NMS_1_21_R3.java

```diff
@@ -1011,7 +1025,17 @@ public final void reloadDataPacks() {
 			this.<MinecraftServer>getMinecraftServer().resources.close();
 			this.<MinecraftServer>getMinecraftServer().resources = serverResources;
 			this.<MinecraftServer>getMinecraftServer().server.syncCommands();
-			this.<MinecraftServer>getMinecraftServer().getPackRepository().setSelected(collection);
+			if (!setSelectedWithBoolean) {
```

Comment:
We could perhaps get rid of `setSelectedWithBoolean` and just do:

```suggestion
			if (minecraftServerSetSelected == null) {
```

If we did do that, it might be easier to read with the if statement flipped around. So rather than saying `if minecraftServerSetSelected does not exist (== null) then don't use it, otherwise alternative` we can have the logic `if minecraftServerSetSelected exists (!= null) then use it, otherwise alternative`.